src/helpers/ci.ts

```diff
@@ -298,7 +328,6 @@ export const getCISpanTags = (): SpanTags | undefined => {
 
     if (GITHUB_BASE_REF) {
       // GITHUB_BASE_REF is defined if it's a pull_request or pull_request_target trigger
-      tags[GIT_BASE_REF] = GITHUB_BASE_REF
```

Comment:
I have removed this tag altogether:
- it was not part of our CI spec (no other library was setting it)
- it was only set for GitHub
- it's value was the same as that of `git.pull_request.base_branch` (set below: `tags[GIT_PULL_REQUEST_BASE_BRANCH]`)